Bottom: 64c551789aa853d05acd75b729034cd785582eac
Top:    d79d5464622c07a1ee7c42051fee5fca02c52207
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-01 10:52:03 -0700

man: List RDMA_PS_IB as a supported port space in rdma_getaddrinfo man page

Signed-off-by: Hal Rosenstock <hal@mellanox.com>
Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/man/rdma_getaddrinfo.3 b/man/rdma_getaddrinfo.3
index 31c233d..86e00ca 100755
--- a/man/rdma_getaddrinfo.3
+++ b/man/rdma_getaddrinfo.3
@@ -55,7 +55,8 @@ are: AF_INET, AF_INET6, and AF_IB.
 Indicates the type of RDMA QP used for communication.  Supported types are:
 IBV_UD (unreliable datagram) and IBV_RC (reliable connected).
 .IP "ai_port_space" 12
-RDMA port space in use.  Supported values are: RDMA_PS_UDP and RDMA_PS_TCP.
+RDMA port space in use.  Supported values are: RDMA_PS_UDP, RDMA_PS_TCP,
+and RDMA_PS_IB.
 .IP "ai_src_len" 12
 The length of the source address referenced by ai_src_addr.  This will be 0
 if an appropriate source address could not be discovered for a given
